index.bs

```diff
@@ -260,6 +265,9 @@ interface ImageCapture {
 
   <dt><dfn dict-member for="PhotoSettings"><code>fillLightMode</code></dfn></dt>
   <dd>This reflects the desired <a>fill light mode</a> (flash) setting.</dd>
+
+  <dt><dfn dict-member for="PhotoSettings"><code>imageFormat</code></dfn></dt>
+  <dd>When the UA returns a {{Blob}} from {{takePhoto()}}, the format and {{Blob/type}} MUST be one of the supported <a>image formats</a> enumerated in {{PhotoCapabilities/imageFormat}}, if any.</dd>
```

Comment:
Sorry, was OOO. 
@jan-ivar I think proper error handling is missing from takePhoto algorithm. IMO, takePhoto should reject promise with `NotSupportedError` if PhotoSettings with unsupported value is provided. 

For this PR, I can align this text with other dictionary members like:
_`"This reflects the desired <a>image format</a> setting."`_

If you and @yellowdoge think that `takePhoto` should reject promise, maybe it would be better to do it in separate PR. What do you think?